addons/binding/org.openhab.binding.kodi/src/main/java/org/openhab/binding/kodi/internal/protocol/KodiConnection.java

```diff
@@ -262,21 +245,29 @@ private void requestPlayerUpdate(int activePlayer, boolean updateMediaType) {
             }
         }
 
+        String thumbnail = "";
+        if (item.has("thumbnail")) {
+            thumbnail = convertToImageUrl(item.get("thumbnail"));
+        }
+
+        String fanart = "";
+        if (item.has("fanart")) {
+            fanart = convertToImageUrl(item.get("fanart"));
+        }
+
         try {
             listener.updateAlbum(album);
             listener.updateTitle(title);
             listener.updateShowTitle(showTitle);
             listener.updateArtist(artist);
+            listener.updateThumbnail(thumbnail);
```

Comment:
Please add clearing those channel values in method `updateState` (lines [L326ff](https://github.com/openhab/openhab2-addons/pull/2304/files#diff-cf13d584f28ada8d7a87e83665490af0R326) if player stops playing.